lib/assessment.rb

```diff
@@ -5,34 +5,50 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
```

Comment:
How do you make these attributes read-only? You can't access them right now from outside the class.